pkg/storaged/storage-controls.jsx

```diff
@@ -169,7 +169,7 @@ export class StorageOnOff extends React.Component {
                         .finally(() => { self.setState({ promise: null }) });
             }
 
-            self.setState({ promise: promise, promise_goal_state: val });
+            self.setState({ promise, promise_goal_state: val });
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-173310-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)